project/webapp/templates/partials/account/utils/representative_card.html

```diff
@@ -10,6 +10,10 @@
                 <div class="text-lato">Party: {{party}}</div>
             </div>
             <div class="subtitle-lato alignment">-</div>
+            <div class="section">
+                <div class="link-lato name"> Votes same: {{ votes.voted_same }}</div>
```

Comment:
Woohoo! This is a lot easier to "grok", as a simple-minded human (me). 

Now, it would also be good to see this as a **percentage** (`votes.voted_same / (votes.voted_same + votes.voted_different)`), since it distills the idea to a single "similarity" number that is widely understood. 

Not saying it has to be part of this PR, but it would be helpful to the end user (and could serve as the basis of a match-making system where representatives are ordered by compatibility with the user -- a sort-of legislative Tinder).